src/view/com/feeds/ProfileFeedgens.tsx

```diff
@@ -85,6 +89,7 @@ export const ProfileFeedgens = React.forwardRef<
     return items
   }, [isError, isEmpty, isFetched, isFetching, data])
 
+  lastItemsState.current = items
```

Comment:
Setting refs during render is not ideal - it will cause React Compiler to deoptimise the component

https://react.dev/learn/referencing-values-with-refs#best-practices-for-refs